include/openssl/ssl.h

```diff
@@ -894,7 +899,9 @@ typedef enum {
     TLS_ST_SW_KEY_UPDATE,
     TLS_ST_CW_KEY_UPDATE,
     TLS_ST_SR_KEY_UPDATE,
-    TLS_ST_CR_KEY_UPDATE
+    TLS_ST_CR_KEY_UPDATE,
+    TLS_ST_EARLY_DATA,
+    TLS_ST_PENDING_EARLY_DATA_END
```

Comment:
it's a bugfix and we'll backport it to 1.1.0 if needed.  this API is broken.  it's just like opacity.  it must be fixed.